internal/scorecard/plugins/olm_tests.go

```diff
@@ -15,14 +15,18 @@
 package scplugins
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"strings"
 
+	"github.com/operator-framework/api/pkg/manifests"
 	schelpers "github.com/operator-framework/operator-sdk/internal/scorecard/helpers"
```

Comment:
where is the "github.com/operator-framework/api/pkg/manifests"? I am not finding it.

Reply:
that is coming from here:  https://github.com/operator-framework/api/tree/master/pkg/manifests  this is the new validation API being used.  That new dependency should be in the go.mod file.